spec/system/partner_system_spec.rb

```diff
@@ -532,6 +532,39 @@
           assert page.has_content? item_category_2.name
         end
       end
+
+      describe 'deleting an existing partner group' do
+        context 'when the partner group has partners' do
+          let!(:first_partner_group) { create(:partner_group, organization: organization) }
+          let!(:first_partner) { create(:partner, partner_group: first_partner_group) }
+          it 'should not display the delete link' do
+            visit partners_path
+            click_on 'Groups'
+            expect(page).to have_content(first_partner_group.name)
```

Comment:
Can you pass an explicit name in here rather than checking an auto-generated one?

Reply:
Done